pkg/atlas/meta.go

```diff
@@ -1,10 +1,20 @@
 package atlas
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 )
 
+// PathToProviderJSON is the path to the json file
+var PathToProviderJSON = "providers.json"
```

Comment:
This should be passed in by the user as an environment variable, marshalled into a meaningful object and then passed into the initialisation of the Atlas client.

Reply:
do we need this field in the client? Could we just call `providersConfig := getEnvOrDefault("PROVIDERS_CONFIG_FILE", "")` from `getProvider` and avoiding quite unnecessary state into the client?